department-of-veterans-affairs/va.gov-cms
# VACMS-5213: Add user admin tab to every Section.

## Description

See #5213. 

## Testing done

Manual

## Screenshots

![Members_of_this_section___Veterans_Affairs](https://user-images.githubusercontent.com/643678/116784314-9dbeaa00-aa61-11eb-87f9-81d7f2f6bd2a.png)

## QA steps

As admin or user admin
* visit /section/veterans-affairs/outreach-hub
* you should see a tab for Section members (/taxonomy/term/7/people)
* visit /admin/people and filter by Outreach Hub
* block some of those editors 
* go back to /taxonomy/term/7/people
* Blocked users should appear separately

As a content publisher
* You shouldn't see a Sections member tab at visit /section/veterans-affairs/outreach-hub
* /taxonomy/term/7/people should be access denied.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
